### ISSUE TITLE
Low Performance for DepthFormer on NYU dataset

### ISSUE THREAD

**Issue by ZaiyiHu** [zhyever/Monocular-Depth-Estimation-Toolbox#98]
Hi, thanks for your great work!

I' been trying to reproduce the results on nyu dataset using depthformer, however it seems to be hard to reproduce the result. The metrics is obviously lower than the most of the papers reported and the visualization looks blurred. I have not idea what could possibly go wrong. Here is my training log. 

Looking for any help!  Thanks again.
[nyu.log](https://github.com/user-attachments/files/16065687/nyu.log)

**Comment by zhyever** [comment]
Hi, sorry for the late reply. The raw config works for 4gpus training as far as I can remember. Please change the schedule, learning rate, etc, based on your experiment settings. Thanks for your interest in my work.
